webapp/src/errorList.tsx

```diff
@@ -29,27 +29,37 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
     }
 
     render() {
+        const showAvailableErrors = this.state.errors.length != 0;
```

Comment:
be defensive and test that errors is defined. In general, we extract the state in const variables:
```
const { errors } = this.state;
```